### PR TITLE
✨ Source Greenhouse - add ```custom fields``` and ```keyed_custom_fields``` to Candidates stream

### PR DESCRIPTION
**Pull request by robertoarnetoli**
## What
The change solves the issue [**Source Greenhouse: Candidates stream does not contains custom fields #29892**](https://github.com/airbytehq/airbyte/issues/29892)

## How
The `candidates` stream schema did not include the `custom_fields` and the `keyed_custom_fields` properties, which have now been added.

## 🚨 User Impact 🚨
Users can now extract `custom_fields` and `keyed_custom_fields`  from the Greenhouse candidates stream.

### CODE REVIEW

File: airbyte-integrations/connectors/source-greenhouse/source_greenhouse/schemas/candidates.json [hunk 1]
```diff
@@ -217,6 +217,14 @@
     },
     "addresses": {
       "type": "array"
+    },
+    "custom_fields": {
+      "properties": {}, 
+      "type": "object"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-greenhouse/source_greenhouse/schemas/candidates.json:223]:
```suggestion
      "additionalProperties": true,
      "type": ["null", "object"]
```

Comment (marcosmarxm) [reply airbyte-integrations/connectors/source-greenhouse/source_greenhouse/schemas/candidates.json:223]: For both fields.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Some comments.

File: airbyte-integrations/connectors/source-greenhouse/source_greenhouse/schemas/candidates.json (hunk 1 above)
Comment (robertoarnetoli) [reply airbyte-integrations/connectors/source-greenhouse/source_greenhouse/schemas/candidates.json]: @marcosmarxm done. thanks

**Comment by robertoarnetoli** [comment]
> Some comments.

Hi @marcosmarxm I have made the suggested changes. Please review.

**Review by sajarin — APPROVED** [review]
Thanks for the contribution! Appreciate you adding these additional fields to this stream.

**Pull request merged**